ui/uihandler.go

```diff
@@ -513,16 +501,24 @@ func Serve(port int, debugpath string, uipath string) {
 		debugPath = ".." + debugpath // TODO: calculate actual path if not same number of directories
 		http.HandleFunc(uiPath, Handler)
 		fmt.Printf("UI starting - visit:\nhttp://localhost:%d%s\n", port, uiPath)
-		logoPath = uiPath + logoURI
-		http.HandleFunc(logoPath, LogoHandler)
-		chartJSPath = uiPath + chartjsURI
-		http.HandleFunc(chartJSPath, ChartJSHandler)
-		// Use relative paths after that (in the html template):
-		logoPath = "./" + logoURI
-		chartJSPath = "./" + chartjsURI
+
 		fetchPath = uiPath + fetchURI
 		http.HandleFunc(fetchPath, FetcherHandler)
 		fhttp.CheckConnectionClosedHeader = true // needed for proxy to avoid errors
 	}
+	logoPath = "../static/img/logo.svg"
+	chartJSPath = "../static/js/Chart.min.js"
+
+	// Serve static contents in the ui/static dir.
+	// We use directory relative to this file to find the static contents,
+	// so no matter where the generate go binary is, the static dir could be found.
+	_, filename, _, ok := runtime.Caller(0)
+	if ok {
+		fs := http.FileServer(http.Dir(path.Join(path.Dir(filename), "static")))
+		http.Handle("/static/", AddCacheControl(http.StripPrefix("/static/", fs)))
```

Comment:
what does the strip prefix do?
does this fileserver logs requests ?(or can we make it do so)

Reply:
from the doc it doesn't seem stripping the same prefix you expect is useful ?

let's leave all the resources rooted under uiPath (so by default `fortio/static/Chart.min.js` instead of ../)